dive_core/capture_data.cpp

```diff
@@ -280,7 +280,7 @@ void MemoryManager::Finalize(bool same_submit_copy_only, bool duplicate_ib_captu
                     DIVE_ASSERT(memory_block.m_va_addr == prev_addr);
 
                     // Use whichever one is bigger and get rid of the smaller one
-                    if (memory_block.m_data_size > temp_memory_blocks.back().m_data_size)
+                    if (memory_block.m_data_size >= temp_memory_blocks.back().m_data_size)
```

Comment:
we have 2 blocks that has the same size and gpu address, but the 2nd one has the most updated content